### PR TITLE
Add the number of new ideas to widget_view event.

### PR DESCRIPTION
**Pull request by asvinb**
## Summary

Addresses issue #4114 

## Checklist

- [x] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/index.js [hunk 1]
```diff
@@ -143,11 +144,19 @@ function DashboardIdeasWidget( props ) {
 		triggerSurvey,
 	] );
 
+	if ( initialTotalNewIdeas === null && newIdeas?.length ) {
```
Comment (aaemnnosttv) [inline assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/index.js:147]: I think we would want to still raise this even if `newIdeas` was an empty array.

**Review by aaemnnosttv — CHANGES_REQUESTED** [review]
Thanks @asvinb – this is almost there, just one small change.

File: assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/index.js (hunk 1 above)
Comment (asvinb) [reply assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/index.js]: Good catch @aaemnnosttv . Updated.

**Review by aaemnnosttv — APPROVED** [review]
LGTM, thanks @asvinb 👍

**Pull request merged**